Packs/FiltersAndTransformers/Scripts/IfElif/IfElif.py

```diff
@@ -0,0 +1,122 @@
+import demistomock as demisto
+from CommonServerPython import *
+from collections.abc import Callable
+from functools import reduce, partial
+import ast
+
+
+class IfElif:
+    operator_functions: dict[type, Callable] = {
+        # comparison operators:
+        ast.Eq: lambda x, y: x == y,
+        ast.NotEq: lambda x, y: x != y,
+        ast.Lt: lambda x, y: x < y,
+        ast.LtE: lambda x, y: x <= y,
+        ast.Gt: lambda x, y: x > y,
+        ast.GtE: lambda x, y: x >= y,
+        ast.In: lambda x, y: x in y,
+        ast.NotIn: lambda x, y: x not in y,
+        # boolean operators:
+        ast.And: lambda x, y: x and y,
+        ast.Or: lambda x, y: x or y,
+        # unary operators:
+        ast.Not: lambda x: not x,
+        ast.USub: lambda x: -x,
+    }
+
+    def __init__(self, value, conditions, flags=None):
+        self.conditions: list
+        self.handle_flags(argToList(flags))
+        self.load_conditions_with_values(conditions, value)
+
+    def handle_flags(self, flags: list):
```

Comment:
lets rename to override_functions_based_on_flags or some name that tells us more what the function does